Modules/ldapcontrol.c

```diff
@@ -343,16 +343,33 @@ encode_assertion_control(PyObject *self, PyObject *args)
         goto endlbl;
     }
 
+    /* XXX: ldap_create() is a nasty and slow hack. It's creating a full blown
+     * LDAP object just to encode assertion controls.
+     */
```

Comment:
I took the liberty to mark the comment with `XXX`.

Reply:
The issue is tracked in https://github.com/python-ldap/python-ldap/issues/86